src/PostgREST/Logger.hs

```diff
@@ -113,3 +120,20 @@ logWithZTime :: LoggerState -> Text -> IO ()
 logWithZTime loggerState txt = do
   zTime <- stateGetZTime loggerState
   hPutStrLn stderr $ toS (formatTime defaultTimeLocale "%d/%b/%Y:%T %z: " zTime) <> txt
+
+logMainQ :: LoggerState -> MainQuery -> IO ()
+logMainQ loggerState MainQuery{mqOpenAPI=(x, y, z),..} =
+  let snipts  = renderSnippet <$> [mqMain, x, y, z]
```

Comment:
Logging the transaction vars query and the pre-req query should just be:
```suggestion
logMainQ :: LoggerState -> MainQuery -> IO ()
logMainQ loggerState MainQuery{mqOpenAPI=(x, y, z),..} =
  let snipts  = renderSnippet <$> [mqMain, x, y, z, mqTxVars, mqPreReq]
```

Can be done on a later PR.